test/06-webresource.test.ts

```diff
@@ -1087,6 +1088,59 @@ describe('06 webresources tests', function () {
 			});
 		},
 	);
+
+	describe('presigned upload', () => {
```

Comment:
The tests can use a good rework here. This is just one simple test to see that the concept works, if the broader backend team agrees on the approach they will be far more sophisticated